ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/config_manager.py

```diff
@@ -109,12 +114,19 @@ def _get_loopback_mappings(self) -> dict:
             # If the host is a VTEP, add the VTEP IP to the mapping
```

Comment:
this comment is now slightly wrong - depending on the source interface we don't add the VTEP IP to the same mapping

Reply:
changed